### ISSUE TITLE
Field::media issue if original image smaller than the thumbnail

### ISSUE THREAD

**Issue by codebymikey** [themosis/framework#665]
Similar to #28.

Stacktrace:
```
themosis.core.js?ver=2.0.1:formatted:34183 Uncaught TypeError: Cannot read property 'url' of undefined
    at t.select (themosis.core.js?ver=2.0.1:formatted:34183)
    at s (backbone.min.js?ver=1.2.3:1)
    at r (backbone.min.js?ver=1.2.3:1)
    at m (backbone.min.js?ver=1.2.3:1)
    at b.Collection.k.trigger (backbone.min.js?ver=1.2.3:1)
    at b.Collection._onModelEvent (backbone.min.js?ver=1.2.3:1)
    at s (backbone.min.js?ver=1.2.3:1)
    at r (backbone.min.js?ver=1.2.3:1)
    at m (backbone.min.js?ver=1.2.3:1)
    at N.d.k.trigger (backbone.min.js?ver=1.2.3:1)
```

Because the image doesn't have a thumbnail, the exception above is thrown.

[Relevant line of code](https://github.com/themosis/framework/blob/fb6a39b/resources/assets/js/src/components/fields/MediaField.tsx#L49). There might be other portions of the app which makes the assumption that a thumbnail size always exists, so it might be worth looking into.

Something along the lines of this should work:
```javascript
const sizes = media.get('sizes');
const size = sizes.thumbnail ? sizes.thumbnail : sizes.full;
if (size) {
    thumbnail = size.url;
}
```

**Comment by codebymikey** [comment]
The image doesn't get uploaded if that's the case. Replicable in the https://github.com/themosis/bookstore/tree/2.0 demo and uploading the small avatar images.

**Comment by jlambe** [comment]
Yup, we probably need to check the code on the collection field as well. Nice catch! Are you up for a PR ? :)

**Comment by jlambe** [comment]
Fixed and available on branch `2.0`. Coming with release `2.0.2`. Thanks for pointing this one out.
